scripts/dotnet.py

```diff
@@ -308,7 +308,7 @@ def install(
 
     if platform != 'win32':
         chmod(dotnetInstallScriptPath, S_IRWXU)
-
+        
```

Comment:
This file only have white spaces changes?

Reply:
Yes. This PR is not ready for the review yet.